CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 - [185](https://github.com/awslabs/amazon-s3-find-and-forget/pull/185): Fix dead
   links to VPC info in docs
+- [186](https://github.com/awslabs/amazon-s3-find-and-forget/pull/186): Fix
+  forget task log sanitiser to handle integer match IDs
```

Comment:
```suggestion
- [186](https://github.com/awslabs/amazon-s3-find-and-forget/pull/186): Fix:
  Solves an issue where the forget phase container could crash when redacting numeric Match IDs from its logs
```

I know this suggestion basically says the same thing, but I think it should be more accessible to operators who may not (want to) know all the internals and are just assessing if a bug affects them.

Reply:
good call